inst/cloudml/cloudml/deploy.R

```diff
@@ -14,13 +7,23 @@ Sys.setenv(R_CONFIG_ACTIVE = config)
 environment <- strsplit(arguments[[3]], "=")[[1]][[2]]
 Sys.setenv(CLOUDML_EXECUTION_ENVIRONMENT = environment)
 
+# install required R packages if we in gcloud
```

Comment:
@kevinushey Note we were installing packages even for `train_local`. Not sure if this was intended, feel free let me know if I should revert this part.